### PR TITLE
/k8s landing refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied refresh as per [design](https://www.figma.com/file/wGMqsc7u32nWdKQQ9Rv8fP/Ubuntu.com-rebranding?node-id=202%3A408&mode=dev) and [doc](https://docs.google.com/document/d/1b018GoKMF3abEFJlUkfAEK2JEhXWICKF4V0dGdTvgJs/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13638.demos.haus/k8s
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8913

### CODE REVIEW

**Comment by Sophie-32** [comment]
@juanruitina - I think you were working on this page? happy to review, but would want background. For example, since the copydoc is different in some places than design

**Comment by mtruj013** [comment]
@Sophie-32 yup, sorry just caught that. I've reassigned 🙂

**Comment by lyubomir-popov** [comment]
+1, I've fixed a few images and some responsive things. one thing to note - anchors need wrapping in paragraphs or other heading tags, otherwise they are not properly spaced.

**Comment by juanruitina** [comment]
Looking good, some minor comments:

- [ ] The logos under "Fully supported Kubernetes clouds" look blurry in my screen:

Screenshot 2024-03-05 at 09 47 09

- [ ] Please add alt text for all logos, including the ones in the hero (Certified Kubernetes, Kubernetes certified service provider) and the ones in the public cloud section.
- [ ] For the logos under "Canonical Kubernetes distributions", please add alt text and put them inside H3 tags (they should act semantically as headings). Please also link the MicroK8s logo to https://microk8s.io/ and the Charmed Kubernetes logo to https://ubuntu.com/kubernetes/charmed-k8s
- [ ] This is an awkward one 🙃 but: let's turn the H2-looking H3s (for instance "Cloud-tailored experience...") into plain paragraphs. They are not headings, they act more like bylines. One exception: turn "Bootstrap your Kubernetes journey" into a paragraph and turn the H3 ("Canonical Kubernetes consulting...") into an H2; that's more meaningful.
- [ ] Consequently, please turn the H4s under "Bootstrap your Kubernetes journey" into H3s.

**Comment by mtruj013** [comment]
@juanruitina pushed the changes, just had a question about the alt text for the hero images. Could you check if they're correct? Wasn't sure if it should be "Canonical Certified Kubernetes" vs what I have now 

cc: @Sophie-32

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -6,409 +6,379 @@
...
+          height="128",
           hi_def=True,
           loading="lazy",
-          attrs={"style": "padding: 30px 20px;"}
           ) | safe
         }}
       </div>
+      <hr />
+      <a class="p-button--positive js-invoke-modal" href="/kubernetes/contact-us">Get in touch</a>
+      <a href="/kubernetes/install">Try it now&nbsp;&rsaquo;</a>
```
Comment (lizzochek) [inline templates/kubernetes/index.html:43]: Should be 'Try Canonical Kubernetes today'

File: templates/kubernetes/index.html [hunk 2]
```diff
@@ -6,409 +6,379 @@
...
-            alt="",
-            width="320",
+  <div class="u-fixed-width">
+    <div class="p-logo-section">
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/3948d6dd-openstack-logo.png",
+            alt="OpenStack",
```
Comment (lizzochek) [inline templates/kubernetes/index.html:253]: Some of the logos differ from the ones in the copy doc. @Sophie-32 are the current logos correct?

File: templates/kubernetes/index.html [hunk 3]
```diff
@@ -6,409 +6,379 @@
...
+    <hr class="p-rule" />
+    <div class="col">
+      <h2 class="p-text--small-caps p-section--shallow">Public cloud Kubernetes<br /> powered by Ubuntu</h2>
+      <p class="p-heading--2">Cloud-tailored experience, optimised kernels, unlimited options</p>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-3">
+          <ul class="p-list--divided">
+            <li class="p-list__item">
```
Comment (lizzochek) [inline templates/kubernetes/index.html:59]: First logo looks too close to the hr, maybe worse adding p-image-wrapper

**Review by lizzochek — APPROVED** [review]

File: templates/kubernetes/index.html (hunk 2 above)
Comment (mtruj013) [reply templates/kubernetes/index.html:253]: Design is the source of truth for images, doc is probably out of date

**Pull request merged**